src/test/java/com/fittracker/fittracker/controller/WeightControllerIntegrationTest.java

```diff
@@ -4,20 +4,24 @@
 import com.fittracker.fittracker.entity.Weight;
 import com.fittracker.fittracker.repository.UserRepository;
 import com.fittracker.fittracker.repository.WeightRepository;
+import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpMethod;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
unused import (also: we should use `com.fasterxml.jackson.databind.ObjectMapper` when we need mapper)